gocse/README.md

```diff
@@ -39,16 +39,16 @@
 
 6. Run the `go run -tags=cse .`. This does the following:
 
-   - Local Key Management Service (KMS)
+   - When using a local master key
 
      - Reads in the local key from `master-key.txt`
 
-   - Remote KMS
+   - When using a Key Management Service (KMS)
```

Comment:
nitpick: for accuracy, this should be Key Management System (https://en.wikipedia.org/wiki/Key_management#Key_management_system). I wonder if we inconsistently refer to it as "key management system" in other places. The AWS KMS product name is Key Management Service but it is an example of a key management system.